src/index.ts

```diff
@@ -345,6 +349,54 @@ class Spanner extends GrpcService {
       [CLOUD_RESOURCE_HEADER]: this.projectFormattedName_,
     };
     this.directedReadOptions = directedReadOptions;
+    this.emulatorHost = emulatorHost;
+  }
+
+  // Helper for instance admin client API calls
+  get_instance_admin_client(this): void {
+    const clientName = 'InstanceAdminClient';
+    if (!this.clients_.has(clientName)) {
+      // add the client created to the clients_ map
+      /* NOTE: when the spanner client is closed, all the clients present in the clients_ map will also be closed as well */
```

Comment:
nit: prefer to stick to one style of comments, and inside a method like this, the most logical is the `// ...` style.